scripts/xtensa-build-all.sh

```diff
@@ -11,7 +11,7 @@ DEFAULT_PLATFORMS=(
     tgl tgl-h
     imx8 imx8x imx8m imx8ulp
     rn rmb
-    mt8186 mt8195
+    mt8186 mt8188 mt8195
```

Comment:
The container is not updated automatically (more container info in #5073). In the mean time try this instead to avoid CI failure like  the one below

```diff
--- a/scripts/xtensa-build-all.sh
+++ b/scripts/xtensa-build-all.sh
@@ -18,11 +18,17 @@ DEFAULT_PLATFORMS=(
 # the -a option for everyone.
 SUPPORTED_PLATFORMS=( "${DEFAULT_PLATFORMS[@]}" )
 
+# Waiting for container work in progress
+SUPPORTED_PLATFORMS=( mt8188  )
+
+
 # Not actually "supported" in the main branch anymore (go to stable-v2.3

```

https://github.com/thesofproject/sof/actions/runs/3692264763/jobs/6250986683

```
    xtensa-mt8188-elf-gcc

  is not a full path and was not found in the PATH.
```